packages/maskbook/src/plugins/Ideamarket/UI/assets/idea-markets-logo.svg

```diff
@@ -0,0 +1 @@
.39,278,734,300.59,738.66,323.51c7.17,35.41,6.94,71.05-3.1,106-6.55,22.8-15.6,44.42-28.47,64.85-14.18,22.51-29.62,44-45.18,65.5-13.29,18.32-25.33,37.34-34.5,57.82S615,660.46,614.76,683c-.23,25.23-.84,50.45-2.89,75.62a119.71,119.71,0,0,1-17.53,53.51c-7.16,11.59-19.18,19.48-31.81,24.55-29.1,11.69-59.1,10.58-89.14,3.58-1.31-.31-2.46-1.3-3.69-2-.92-3.89-2.47-6.47-6.79-3.23-4.43-.86-7.86-3.61-11.43-6.1.65-.5,2-1.25,1.87-1.47-2.94-7.82-6.56-15-7.47-24S446.12,788.43,455.51,785.34Zm1.32,9c-1.64-.78-2.16-.18-2.23.72-.08,1.22.43,2.3,1.77,2.39.6,0,1.25-.58,1.87-.91Zm-3.53,22.61c.48.21,1.06.68,1.4.55s.44-.88.65-1.35c-.47-.21-1.05-.68-1.39-.55S453.51,816.46,453.3,816.93Zm10.42,9.85-1.28.75c.43.24.86.67,1.28.67s.86-.42,1.29-.66Zm41.5-402.06c-15.12-2.66-23.57-10.91-33.43-21.7-8.26-9-18.19-15.31-28.08-21.7-8.87-5.72-24.09-2.85-32.33,4.1-13,11-15.63,26.25-18.2,41.74-3.29,19.82-6.89,39.84-16.51,57.6-8.36,15.42-16.06,17.9-30.63,7.88-13.22-9.08-22.3-22.05-27.73-36.56-11.65-31.2-14.74-63.91-12.89-97.05,1.32-23.63,6.21-47,15.32-68.44a170.28,170.28,0,0,1,42.58-60.44c20.27-18.34,43.34-30.56,68.48-39.06,19.8-6.7,40.87-8.49,61.75-10.09,1.69-5.84,7-1.51,10.22-3.75-1.85-1.66-4.1-3-2.59-6.42,3.72-8.46,1-16.92-.85-25.31-28,1.79-41.58,3.57-61.9,8.51-22,5.35-43.68,11.85-62.81,24.32-10.84,7.07-18.68,11.21-31.71,22.43-11,10.3-17.39,16.16-27.15,30.37-10.39,15.47-20.32,30.53-27.85,47.75-13.34,30.49-18.45,62.23-18.93,95a264.89,264.89,0,0,0,2.14,40.69,234.14,234.14,0,0,0,18.31,62.44c11.2,25.25,28.81,44.12,54.55,53.79,16.63,6.26,33.55,3,46.86-9.17,14.37-13.11,22.5-30.5,26.44-49.35,3.52-16.9,7.6-33.66,11.33-50.5,1.74-7.89,4.87-8,9.44-2.66,9.05,10.66,20.09,18.91,31.46,26.78,10,6.93,20.37,13,30.74,14.06m0,0c4.13.71,29-2.64,39.45-12.19,5.4-4.93,10.53-10.18,15.48-15.56,14.74-16,22.63-36.45,34.7-54.21,3.21-4.72,5.4-4.71,9.37-2.26,5.3,3.27,10.75,6.3,16,9.67,4.72,3,6.68,2.85,6.64-3.63q-.33-43.39.07-86.78c.08-7.54-1.79-8-8-4.6-18.3,10.08-29.05,15.54-44.6,24.36-13.21,7.58-20.9,12.09-34.9,21.23,8.21,5.18,15.93,10.24,23.84,15,3.77,2.26,5.61,4.41,2.67,8.69-10.29,21-20.39,37.2-30.85,50.28-6.13,7.66-13.85,14.73-25.91,14.73M445.09,741.05c-.46-21.55-.88-43.12-1.86-64.65-.84-18.43-2.24-37.15-11.62-53.47-7.76-13.48-14.84-28.37-32.09-33.15-6.35-1.76-16.75,1.68-20,7.34-3.41,6-1.85,14.27,3.83,19.23,12.68,11.07,19.93,24.87,22.36,41.58,2.22,15.2,2.44,30.46,3.28,45.73,1.48,27.06,1.11,54.27,9.33,80.72,5.94,19.1,17.4,33.21,33.13,44.59.65-.5,2-1.25,1.87-1.47-2.94-7.82-6.56-15-7.47-24s.24-15.12,9.63-18.21C448.57,771.39,445.42,756.59,445.09,741.05ZM469.7,838.3c-.92-3.89-2.47-6.47-6.79-3.23C464.68,837.2,467.23,837.66,469.7,838.3Zm-8.31-44.69c.81,1.65-.35,4.11,2.11,5,.06,0,.73-1.73,1.11-2.66ZM454.6,795c-.08,1.22.43,2.3,1.77,2.39.6,0,1.25-.58,1.87-.91l-1.41-2.2C455.19,793.54,454.67,794.14,454.6,795Zm.1,22.44c.34-.14.44-.88.65-1.35-.47-.21-1.05-.68-1.39-.55s-.45.88-.66,1.35C453.78,817.14,454.36,817.61,454.7,817.48Zm7.74,10c.43.24.86.67,1.28.67s.86-.42,1.29-.66l-1.29-.76Z" transform="translate(-269.94 -145.52)"/></svg>
```

Comment:
Format it.

Reply:
File wasn't used anymore, svg:s are inside assets/index.tsx, so I deleted it.